### ISSUE TITLE
Media text block doesn't show upload errors

### ISSUE THREAD

**Issue by thesquaremedia** [WordPress/gutenberg#15684]
**Describe the bug**
When I upload an image that would throw an upload error such as an image bigger than the allowed size. The Block won't display any errors.

**To reproduce**
1. Go to 'Page/Post and add the media and text block'
2. Click on 'Upload Image'
3. Upload an image larger than the allowed size.'
4. Notice that the block wont do anything and wont display an error like other image blocks do.

**Expected behavior**
We should be able to see an upload related error like in the other image blocks.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Browser Safari
 - Version Mojave

**Additional context**
- Wordpress 5.1.1

**Comment by draganescu** [comment]
Related: #19156 19156

**Comment by mapk** [comment]
I just tested this out with WordPress 5.3 on Firefox 71.0 and got the error messages as expected. Perhaps this has already been resolved. 

**Image block**

![Screen Shot 2019-12-27 at 2 12 02 PM](https://user-images.githubusercontent.com/617986/71534397-7119e180-28b3-11ea-8e90-186fc0723f00.png)

**Media + Text block**

![Screen Shot 2019-12-27 at 2 12 46 PM](https://user-images.githubusercontent.com/617986/71534403-75de9580-28b3-11ea-996c-872048c0e042.png)

I'll close this issue for now keeping in mind there is discussion about where we display these errors in #19156.
